src/bond-sdk-cards.js

```diff
@@ -103,7 +103,14 @@ class BondCards {
     return new Promise((resolve, reject) => {
       const newIframe = this.internalShow.request(requestParams);
       if (newIframe) {
-        resolve(newIframe.render(htmlSelector, css));
+        const iframe = newIframe.render(htmlSelector, css);
+        const subscribe = (event) => {
+          if(event.data.messageName === "update" && event.data.payload.revealed === true){
```

Comment:
With the inconsistent spacing after "if" it seems like you don't have Prettier formatting your code in your IDE? We'd like all the devs to be consistent and we use Prettier and a customized set of AirBnb's ESLint rules to make sure everyone is doing the same syntax and code-style. Let me know if you'd like help setting those up. There's a confluence page with instructions too.